src/Wallabag/CoreBundle/Command/CleanDuplicatesCommand.php

```diff
@@ -0,0 +1,111 @@
+<?php
+
+namespace Wallabag\CoreBundle\Command;
+
+use Doctrine\ORM\NoResultException;
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Wallabag\CoreBundle\Entity\Entry;
+use Wallabag\UserBundle\Entity\User;
+
+class ExportCommand extends ContainerAwareCommand
+{
+	/** @var OutputInterface */
+	protected $output;
+
+	protected function configure()
+	{
+		$this
+			->setName('wallabag:cleanduplicates')
+			->setDescription('Cleans the database for duplicates')
+			->setHelp('This command helps you to clean your articles list in case of duplicates')
+			->addArgument(
+				'username',
+				InputArgument::OPTIONAL,
+				'User to clean'
+			);
+	}
+
+	protected function execute(InputInterface $input, OutputInterface $output)
+	{
+		$this->output = $output;
+
+		$username = $input->getArgument('username');
+
+		if ($username) {
+			try {
+				$user = $this->getUser($username);
+				$this->cleanDuplicates($user);
+			} catch (NoResultException $e) {
+				$output->writeln(sprintf('<error>User "%s" not found.</error>', $input->getArgument('username')));
+
+				return 1;
+			}
+		} else {
+			$users = $this->getDoctrine()->getRepository('WallabagUserBundle:User')->findAll();
+
+			$output->write(sprintf('Cleaning through %d user accounts', count($users)));
+
+			foreach ($users as $user) {
+				$this->cleanDuplicates($user);
```

Comment:
Maybe output which user you are working one. It'll be easier to debug and check the progress of the command.